README.md

```diff
@@ -1 +1,5 @@
 # 🚁 < 🛸
+
+cd post-conveter-script
+npm install or yarn install
```

Comment:
really should stick with one or the other (preferably yarn)

Reply:
Okay, I'll stick to yarn.